go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/nukosuke/go-zendesk
+module github.com/chrisjoyce911/go-zendesk
```

Comment:
please revert the namespace for this repo

Reply:
Done